aiinfra-cluster/modules/vm-instance-group/main.tf

```diff
@@ -25,6 +25,12 @@ locals {
   enable_gvnic  = var.bandwidth_tier != "not_enabled"
   enable_tier_1 = var.bandwidth_tier == "tier_1_enabled"
 
+  is_debian_11_image = (var.instance_image.name != null && length(regexall("debian-11", var.instance_image.name)) > 0) || (var.instance_image.family != null && length(regexall("debian-11", var.instance_image.family)) > 0)
```

Comment:
Should we support non debian image families (e.g. centos)

Reply:
Currently we are limiting to dlvm and I didn't see any dlvm centos image. As we expand out of DLVM, we can start supporting centos, but that is a separate project by itself.

```
centos-7-v20230306                                      centos-cloud         centos-7                                       READY
centos-stream-8-v20230306                               centos-cloud         centos-stream-8                                READY
centos-stream-9-v20230306                               centos-cloud         centos-stream-9                                READY
```